OTRExporter/OTRExporter/Main.cpp

```diff
@@ -63,13 +63,29 @@ static void ExporterProgramEnd()
 {
 	if (Globals::Instance->fileMode == ZFileMode::ExtractDirectory)
 	{
-		printf("Generating OTR Archive...\n");
-		otrArchive = Ship::Archive::CreateArchive(otrFileName, 40000);
+    printf("Creating version file...\n");
 
-		for (auto item : files)
-		{
-			auto fileData = item.second;
-			otrArchive->AddFile(item.first, (uintptr_t)fileData.data(), fileData.size());
+    // Get crc from rom
+    std::string romPath = Globals::Instance->baseRomPath.string();
+    std::vector<uint8_t> romData = File::ReadAllBytes(romPath);
```

Comment:
Not a fan of reading the entire rom into memory again just to get the checksum from the header. But might be fine for now.

Reply:
Yeah it's overkill but there's no function for reading only parts of the rom in [File.h](https://github.com/HarbourMasters/Shipwright/blob/e2e0a070eab85ea6c7eeb32db67b08321bdfd929/ZAPDTR/ZAPDUtils/Utils/File.h), only for reading all of it. I could create it if you think it would be useful or, if we want to avoid changing/adding to ZAPD too much, I could write a couple lines just to do it here.